src/components/share-structure/array-move.ts

```diff
@@ -0,0 +1,57 @@
+/** array-move version 4.0
```

Comment:
No more library, except it's in a file now

Reply:
OK.

In the future, I'd prefer to see this in the src/utils/ file (SnakeCase), like the other standalone functions we have. (This repo is not a great example but see https://github.com/bcgov/business-create-ui/tree/main/src/utils.)

Reply:
I'm flexible on this file's location -- as this is a repo of "standalone" components, I do appreciate having this function/file in the present folder.

I'm thinking ahead about "parent UIs" that might import this as a package, or could use it from a (shared, maybe) utils folder.